docs/cookbook/html-conversion.md

```diff
@@ -0,0 +1,980 @@
+
+<AposCodeBlock>
+::: v-pre
+
+```django
+{% extends 'layout.html' %}
+
+{% import 'fragments/header.html' as header %}
+
+{% block beforeMain %}
+  {{ super() }}
+  {% render header.headerArea(data.page) %}
+{% endblock %}
+
+{% block main %}
+  <!-- Main Content-->
+  <div class="container px-4 px-lg-5">
+    <div class="row gx-4 gx-lg-5 justify-content-center">
+      <div class="col-md-10 col-lg-8 col-xl-7">
+        {% for piece in data.pieces %}
+          <!-- Post Preview -->
+          <div class="post-preview">
+            <a href="{{ piece._url }}">
+              <h2 class="post-title">{{ piece.heading }}</h2>
+              <h3 class="post-subtitle">{{ piece.subheading }}</h3>
+            </a>
+            <p class="post-meta"> Posted by {{ piece.author }} on {{ piece.publicationDate | date('MMMM d, YYYY') }}</p>
+          </div>
+          <!-- Divider-->
+          <hr class="my-4" />
+        {% endfor %}
+        <!-- Pager-->
+        <div class="d-flex mb-4">
+          {% if data.currentPage > 1 %}
+          <a href="{{ data.slug }}?page={{data.currentPage - 1}}" class="btn btn-primary text-uppercase me-auto">newer posts</a>
+          {% endif %}
+          {% if data.totalPages > data.currentPage %}
+          <a href="{{ data.slug }}?page={{data.currentPage + 1}}" class="btn btn-primary text-uppercase ms-auto">older posts</a>
+          {% endif %}
+        </div>
+      </div>
+    </div>
+  </div>
+
+{% endblock %}
+```
+:::
+
+<template v-slot:caption>
+modules/blog-page/index.html
+</template>
+</AposCodeBlock>
+
+Focusing on the `for` loop in the code. We are stepping through all of the articles returned in `data.pieces` and outputing the relevant data. Again, since we specified a `perPage` value of `5` in the options, this will return the five newest blog articles. This can be further configured within the `blog-page` module options, for example with the [`sort` option](https://v3.docs.apostrophecms.org/reference/modules/piece-type.html#sort).
+
+The `Pager` section is expanded to conditionally show newer and older blog articles, unlike the original template, which only shows older articles. Within the section, we are taking advantage of some additional data that is being delivered to the `index.html` page. Within the `data` payload are `data.currentPage` and `data.totalPages`. The `data.totalPages` is how many individual data sets are present for the particular piece type if divided into groups based on the `perPage` option (the default is 10).
+
+By default, we are showing the newest blog articles first. Therefore, if the `data.currentPage` is equal to `1` then we shouldn't display the button to load newer articles. If we are on any other page we want the button displayed, with a URL that adds a query to go to the previous page -
+::: v-pre
+(`{{data.currentPage - 1}}`).
```

Comment:
```suggestion
`{{data.currentPage - 1}}`.
```

Are the parens necessary?

Reply:
I guess not since it is being bumped onto its own line by the ::: pre. Changed.